lab_6_pipeline/pipeline.py

```diff
@@ -3,11 +3,38 @@
 """
 from pathlib import Path
 from typing import List
+import re
+from pymystem3 import Mystem
 
 from core_utils.article.article import SentenceProtocol
 from core_utils.article.ud import OpencorporaTagProtocol, TagConverter
+from core_utils.article.io import from_raw, from_meta, to_cleaned
+from core_utils.constants import ASSETS_PATH
+from core_utils.article.article import Article, split_by_sentence
 
 
+class FileNotFoundError(Exception):
+    """
+    Raised when file does not exist
+    """
+
+
+class NotADirectoryError(Exception):
```

Comment:
same, check if you have shadowed any more built-in exceptions

Reply:
done